ui/src/app/settings/views/Configuration/DeployForm/DeployForm.tsx

```diff
@@ -7,10 +7,12 @@ import FormikForm from "app/base/components/FormikForm";
 import DeployFormFields from "app/settings/views/Configuration/DeployFormFields";
 import { actions as configActions } from "app/store/config";
 import configSelectors from "app/store/config/selectors";
+import { timeSpanToMinutes } from "app/utils";
 
 const DeploySchema = Yup.object().shape({
   default_osystem: Yup.string(),
   commissioning_distro_series: Yup.string(),
+  hardware_sync_interval: Yup.number().min(1),
```

Comment:
Might be nicer to provide an error message as the second argument to `min()` so it's not `"hardware_sync_interval must be greater than or equal to 1"`

Reply:
Updated and added a test for this message.